pages/downloads.html

```diff
@@ -34,7 +34,7 @@ <h3>FMI for Model Exchange and Co-Simulation</h3>
   </button>
   <div class="dropdown-menu dropdown-menu-right">
     <a class="dropdown-item" href="https://github.com/modelica/fmi-standard/releases/download/v2.0.1/FMI-Specification-2.0.1.pdf"><i class="fa fa-file mr-2"></i>Specification Only 2.0.1</a>
-    <a class="dropdown-item" href="https://svn.modelica.org/fmi/branches/public/specifications/v2.0/FMI_for_ModelExchange_and_CoSimulation_v2.0.pdf"><i class="fa fa-file mr-2"></i>Specification Only 2.0</a>
+    <a class="dropdown-item" href="https://github.com/modelica/fmi-standard.org/tree/master/assets/FMI2.0/FMI_for_ModelExchange_and_CoSimulation_v2.0.pdf"><i class="fa fa-file mr-2"></i>Specification Only 2.0</a>
```

Comment:
Why not use the URL it will end up at? You're building the webpage, right? So you should be able to link to fmi-standard.org